AndroidTrainingModule/src/com/cyscorpions/dalejulian/androidtrainingmodule/DatePickerFragment.java

```diff
@@ -0,0 +1,87 @@
+package com.cyscorpions.dalejulian.androidtrainingmodule;
+
+import java.util.Calendar;
+import java.util.Date;
+import java.util.GregorianCalendar;
+
+import android.app.Activity;
+import android.app.AlertDialog;
+import android.app.Dialog;
+import android.content.DialogInterface;
+import android.content.Intent;
+import android.os.Bundle;
+import android.support.v4.app.DialogFragment;
+import android.view.View;
+import android.widget.DatePicker;
+import android.widget.DatePicker.OnDateChangedListener;
+
+public class DatePickerFragment extends DialogFragment {
+	
+	public static final String EXTRA_DATE = "DatePickerFragment.DATE";
+	private Date mDate = new Date();
+	
+	public interface ReceiveDateDialogListener{
+		void onFinishEditDialog(Date date);
+	}
+	
```

Comment:
Please format (Ctrl+Shift+F) this file to remove whitespaces and arrange its spacing.